build_for_tests.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
```

Comment:
```suggestion

set -e

```

I'd expect this script to halt in case `make` or `make install` fails for some reason. This also applies for the other queries down below.